### ISSUE TITLE
Not possible to reproduce results

### ISSUE THREAD

**Issue by osmanmusa** [makbari7/DSSLIC#4]
Dear All,

Unfortunately I am not possible to reproduce results shown in the linked paper, both for pre-trained network and for the self-trained network.

With the pre-trained networks I get the same results as in the paper on the ADE20K and Kodak datasets. On the other hand, on cityscapes, the results I get are the same for the DSSLIC network. However, the results for classical codecs are much better than what is presented in the paper.

When I train the networks myself, the results are far from those in the paper. Even if I continue with training of the pre-trained network with a "super small" learning rate for a few epochs, it seems that the performance of the DSSLIC network is drastically worsened by this small perturbations of the network parameters.

As this strange behaviour of the DSSLIC codec kept me puzzled for a few months, any tips that could help me reproduce results from the paper would be much appreciated. This would also be beneficial for the reputation of the paper, as some authors started reporting worse DSSLIC's results in their papers.
